pages_builder/pages/external-link.yml

```diff
@@ -4,3 +4,4 @@ examples:
   -
     link: https://www.example.com'
     text: Example
+    newwindow: true
```

Comment:
Turns out it doesn't support markdown, but it does support html. I've updated link to actually be a link.